src/routes/(console)/organization-[organization]/billing/planSummary.svelte

```diff
@@ -85,7 +85,7 @@
                                         <h5 class="body-text-2 u-stretch">Additional members</h5>
                                         <div>
                                             {formatCurrency(
-                                                extraMembers * currentPlan.addons.member.price
+                                                extraMembers * currentPlan?.addons?.member?.price
```

Comment:
I think in this case `formatCurrency` might return `NaN`... i think we need to do something like `extraMembers * (currentPlan?.addons?.member?.price ?? 0)`

Reply:
I've changed it!